crates/topos-tce-api/src/runtime/mod.rs

```diff
@@ -346,7 +349,7 @@ impl Runtime {
                                         positions: vec![TargetStreamPosition {
                                             target_subnet_id,
                                             source_subnet_id,
-                                            position: position.0,
+                                            position: *position,
```

Comment:
This is the `Deref` impl in action yeah? I guess my concern is that when a reader sees this they might think that `position` is a pointer to something, which isn't really the case. I think `position.0` is probably more explicit.

Reply:
It's incompatible with the fact that the `u64` is not `private`